guide/start/running.md

```diff
@@ -134,13 +134,15 @@ Extract this zip file to a directory on your computer such as `c:\Program Files\
 
 ---
 
-It is not necessary at this time, but depending on what you are going to do, 
-you may wish to set up other configuration options first:
+By default, no authentication is required and the web-console will listen on all network interfaces.
+For a production system, or if Apache Brooklyn is publicly reachable, it is strongly recommended 
```

Comment:
Perhaps add these justifications to the files above which also recommend the use of security.